Description: Fix spelling errors in manpage
Origin: vendor
Author: Salvatore Bonaccorso <carnil@debian.org>
Last-Update: 2016-01-02
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=110845
Bug: https://rt.cpan.org/Ticket/Display.html?id=110845

--- a/lib/Sys/Virt.pm
+++ b/lib/Sys/Virt.pm
@@ -1494,7 +1494,7 @@ constants described later in this docume
 a subroutine reference that will receive the events.
 
 All callbacks receive a C<Sys::Virt> connection as the first parameter
-and a C<Sys::Virt::Domain> object indiciating the domain on which the
+and a C<Sys::Virt::Domain> object indicating the domain on which the
 event occurred as the second parameter. Subsequent parameters vary
 according to the event type
 
@@ -1558,7 +1558,7 @@ constants described later in this docume
 a subroutine reference that will receive the events.
 
 All callbacks receive a C<Sys::Virt> connection as the first parameter
-and a C<Sys::Virt::Network> object indiciating the network on which the
+and a C<Sys::Virt::Network> object indicating the network on which the
 event occurred as the second parameter. Subsequent parameters vary
 according to the event type
 
